src/map/utils/mobutils.cpp

```diff
@@ -100,25 +100,28 @@ namespace mobutils
                 return battleutils::GetMaxSkill(SKILL_THROWING, JOB_MNK, mlvl); // E Skill (5)
         }
 
-        ShowError("mobutils::GetBase rank (%d) is out of bounds for mob (%u) ", rank, PMob->id);
+        ShowError("mobutils::GetSkill rank (%d) is out of bounds for mob (%u) ", rank, PMob->id);
         return 0;
     }
 
     uint16 GetMagicEvasion(CMobEntity* PMob)
     {
         uint8 mEvaRank = PMob->evaRank;
-        return GetBase(PMob, mEvaRank);
+        return GetSkill(PMob, mEvaRank);
     }
 
     /************************************************************************
      *                                                                       *
-     *  Base value for defense                       *
+     *  Base value for defense and evasion                                   *
      *                                                                       *
      ************************************************************************/
 
-    uint16 GetDefense(CMobEntity* PMob, uint8 rank)
+    uint16 GetBase(CMobEntity* PMob, uint8 rank)
```

Comment:
This is gonna get confusing.

I think we could name them `GetBaseSkill` and `GetBaseDefEva` or something along those lines, so we know what we are doing.